assessment2.css

```diff
@@ -0,0 +1,7 @@
+img{
```

Comment:
CSS is definitely more needed in this case as the page has background and logo as main features. Text tags are on point as they include h1, h3, and p tags as indicated in the rubric. My guess is that you ran out of time after working on js logics. Please re-visit css part of your assessment, and complete it. CSS is an important skill as a developer, and more practice is always better.